providers/home.rb

```diff
@@ -0,0 +1,46 @@
+require 'chef/mixin/shell_out'
+include Chef::Mixin::ShellOut
+
+action :create do
+  task_name = "create_#{@new_resource.name}_home"
+  user_home = ::File.join(ENV['SYSTEMDRIVE'], 'Users', @new_resource.name)
+    .gsub(::File::SEPARATOR, ::File::ALT_SEPARATOR)
+
+  # Run whoami as scheduled task to get LocalServer group privilege:
+  # SeAssignPrimaryTokenPrivilege (Replace a process-level token)
+  if task_query(task_name).empty?
+    cmd =  "schtasks /Create /TN \"#{task_name}\" "
+    cmd += "/SC once /SD \"01/01/2003\" /ST \"00:00\" /TR \"whoami.exe\" "
+    cmd += "/RU \"#{@new_resource.name}\" /RP \"#{@new_resource.password}\" "
```

Comment:
As far as I know, all that does is prevent chef from reporting the sensitive information. I'm not really sure what the right thing to do here is. I'll ping people, see what they think